pkg/scalers/azure_monitor_scaler.go

```diff
@@ -148,6 +149,24 @@ func parseAzureMonitorMetadata(config *ScalerConfig) (*azureMonitorMetadata, err
 
 	meta.scalerIndex = config.ScalerIndex
 
+	azureResourceManagerEndpointProvider := func(env az.Environment) (string, error) {
+		return env.ResourceManagerEndpoint, nil
+	}
+	azureResourceManagerEndpoint, err := azure.ParseEnvironmentProperty(config.TriggerMetadata, "azureResourceManagerEndpoint", azureResourceManagerEndpointProvider)
```

Comment:
Should we extract this function and place in azure package? activeDirectoryEndpointProvider is used in other scalers, IDK if this can be the same for others